criu/files-reg.c

```diff
@@ -1604,15 +1614,16 @@ static int store_validation_data_build_id(RegFileEntry *rfe, int lfd,
 	if (!build_id || build_id_size == -1)
 		return 0;
 
-	rfe->build_id = xmalloc(round_up(build_id_size, sizeof(uint32_t)));
+	allocated_size = round_up(build_id_size, sizeof(uint32_t));
+	rfe->build_id = xmalloc(allocated_size);
```

Comment:
should we use xzalloc here? allocated_size can be bigger than build_id_size, but memcpy is called for build_id_size only.

Reply:
Yeah, you're right. I've made the change.